src/towncrier/test/test_create.py

```diff
@@ -182,3 +182,22 @@ def test_file_exists(self):
 
         self.assertEqual(type(result.exception), SystemExit)
         self.assertIn("123.feature.rst already exists", result.output)
+
+    def test_create_unlinked_fragment(self):
+        """Ensure a random hash is added to unlinked fragment files."""
+        runner = CliRunner()
```

Comment:
I prefer to write the test docstrings as test scenarios.

"ensure" is kind of redendand in the context of a test.
We always write tests to ensure something.

No need to be too specific in the docstring.
A random is good enough... it can be hex, but maybe later we want to have some random names (like docker does :)

```suggestion
    def test_create_unlinked_fragment(self):
        """
        When called for a ticket ID with value C{+}, the create CLI
        automatically expands the new file to a random value.
        """
        runner = CliRunner()
```

Reply:
Done